webpack.config.ts

```diff
@@ -84,10 +83,12 @@ const config: Configuration = {
     new CopyPlugin({
       patterns: [{ from: '../locales', to: '../dist/locales' }],
     }),
-    new DefinePlugin({
-      'process.env.DATA_SOURCE': JSON.stringify('mock'),
-      'process.env.BRAND_TYPE': JSON.stringify('Konveyor'),
-    }) as unknown as WebpackPluginInstance,
+    new EnvironmentPlugin({
+      DATA_SOURCE: 'remote',
```

Comment:
1.  at this point we can start reading the env var and only if it's missing fallback to some hardcoded mode
2. "mock" seems safer default but it's up to you

Reply:
OK. I see EnvironmentPlugin works differently then DefinePlugin. Please ignore.